spec/lib/api/2.0/config-spec.js

```diff
@@ -6,22 +6,22 @@
 describe('Http.Api.Config v2.0', function () {
     var configuration;
     before('start HTTP server', function () {
-        this.timeout(5000);
-        return helper.startServer();
-    });
-
-    beforeEach('set up mocks', function () {
-        configuration = helper.injector.get('Services.Configuration');
-        sinon.stub(configuration, 'set').returns(configuration);
-        sinon.stub(configuration, 'getAll').returns({});
+        this.timeout(10000);
```

Comment:
@brianparry not that I saw, but I figured I would just increase the timeout since I was in there to decrease the likelihood of a failure because of a timeout.